drivers/entropy/Kconfig.sam0

```diff
@@ -0,0 +1,14 @@
+# Kconfig.sam - Atmel SAM0 TRNG configuration
+#
+# Copyright (c) 2019 Benjamin Valentin
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ENTROPY_SAM0_RNG
```

Comment:
Better to `config` than `menuconfig` here. See https://github.com/zephyrproject-rtos/zephyr/pull/20005.